tests/python/gpu/test_gluon_model_zoo_gpu.py

```diff
@@ -37,11 +37,11 @@ def download_data():
     return mx.test_utils.download(
         'http://data.mxnet.io/data/val-5k-256.rec', VAL_DATA)
 
-@unittest.skip("test fails intermittently. temporarily disabled.")
 @with_seed()
 def test_inference():
     all_models = ['resnet50_v1', 'vgg19_bn', 'alexnet', #'inceptionv3',
                   'densenet201', 'squeezenet1.0', 'mobilenet0.25']
+    mx.random.seed(2)
```

Comment:
please don't use mx.random.seed due to the introduction of the @with_seed() decorator

Reply:
Does with_seed() set the same seed for each run?

Reply:
Why would you want to set it the same for each run?